mmjstool/src/i18n_handlers.js

```diff
@@ -199,3 +199,67 @@ export function i18nSplit(argv) {
         }
     });
 }
+
+export function i18nCleanEmpty(argv) {
+    const wCode = i18nCleanEmptyWebapp(argv);
+    const mCode = i18nCleanEmptyMobile(argv);
+    process.exit(wCode || mCode);
+}
+
+export function i18nCleanEmptyWebapp(argv) {
+    const webappDir = argv['webapp-dir'];
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const fPath = path.join(webappDir, 'i18n');
+    return cleanAll(fPath, dryRun, check);
+}
+
+export function i18nCleanEmptyMobile(argv) {
+    const mobileDir = argv['mobile-dir'];
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const fPath = path.join(mobileDir, 'assets', 'base', 'i18n');
+    return cleanAll(fPath, dryRun, check);
+}
+
+function cleanAll(filePath, dryRun, check) {
+    const files = fs.readdirSync(filePath);
+    let results = '';
+    for (const file of files) {
+        if (file.split('.').pop() !== 'json' || file === 'en.json') {
+            continue;
+        }
+        const result = removeItems(filePath, file, dryRun, check);
+        results += result;
```

Comment:
results can be undefined. so you need to check for that before appending

Reply:
But I set it in line 227? I think I don't get javascript.

Reply:
basically `removeItems`  in  line 251 is retuning undefined.

Reply:
But it is the reviver the JSON.parse(text[, reviver]) that is returning undefined only when the translation value is empty, so that the property gets deleted from the object. 
> If the reviver function returns undefined (or returns no value, for example, if execution falls off the end of the function), the property is deleted from the object. 
Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse

I basically stole this info there and applied it in this way. Not really sure it is intended for deletion. eslint wasn't all too happy about it, that's why I had to ignore the rule (`// eslint-disable-line no-undefined`).